README.md

```diff
@@ -0,0 +1,5 @@
+# Humanizing Code Review
+
+## Step 1: Add Humans
+
+<img src=https://78.media.tumblr.com/a2d473b52d40254a41a3a95cec4c2632/tumblr_oytv4pTbOj1vbcnq8o1_500.gif />
```

Comment:
Use different image.

Reply:
You should use a different image.

Reply:
We should use a different image.

Reply:
The readme should use a different image.

Reply:
The readme should use a different image due to copyright control issues. More details [are available here](https://www.youtube.com/watch?v=dQw4w9WgXcQ).

Use [this site](https://www.youtube.com/watch?v=dQw4w9WgXcQ) to find a more suitable source.

Reply:
What's the benefit of using this image of Homer Simpson? Do we have any concerns related to copyright?

What are some other other image choices that may be more suitable here?